docs/source/releases/v1_10_0_dev0.rst

```diff
@@ -37,10 +39,6 @@ Add a validator for YAML-based plugins.
     created: /geoips/schema/__init__.py
     created: /geoips/schema/plugin_validator.py
 
-Add a schema for validating `Boundaries` plugins.
```

Comment:
This was duplicated, other one is still there:

https://github.com/NRLMMD-GEOIPS/geoips/blob/122-convert-yaml-files-for-product_params-interface/docs/source/releases/v1_10_0_dev0.rst#add-a-schema-for-validating-boundaries-plugins